docs/pages/management/admin/labels.mdx

```diff
@@ -41,7 +43,7 @@ but filter and limit access to the servers based on their AWS region.
 
 (!docs/pages/includes/tctl.mdx!)
 
-## Step 1/3. Install Teleport
+### Install Teleport
```

Comment:
I'm curious as to why we're turning these "Step ..." H2s into H3s of the Prerequisites section. I would keep them as they were and add a `Step 4/4.` for resource-based labels. Otherwise, the document structure is a little jarring to me.

Reply:
This was a response to [this comment](https://github.com/gravitational/teleport/pull/34475#discussion_r1392566178) (although making them H3s was just a mistake on my part). I've updated the docs with a hybrid approach that preserves the structure in the prereqs but allows the label sections to be done in any order, let me know what you think.

Reply:
Perfect, thanks!